tests/file.rs

```diff
@@ -83,8 +109,16 @@ async fn read_write_delete_file() -> Result<()> {
     // second file to write
     let (_sk, pk) = generate_keypair(&mut thread_rng());
 
-    // info!("Write Delete:: Writing file if not exists in order to test delete");
-    let blake3_hash = write_file(&Secp256k1PubKey(pk), &file_bytes).await.is_ok();
+    let file_bytes = fs::read("tests/samples/cat.gif")?;
```

Comment:
do we need to shadow "file_bytes" here?

Reply:
Since I no longer rewrite it since I renamed file_stream, you're probably right. I'll be sure to fix that.